installer/bundle/bundle_skel_Linux.sh

```diff
@@ -702,29 +581,25 @@ case "$installMode" in
 
     I)
         echo "Installing cross-platform agent ..."
-        if [ "$PLATFORM" = "SunOS" ]; then
-            check_if_pkg_is_installed scx
-            if [ $? -eq 0 ]; then
-                echo "ERROR: SCX package is already installed"
-                cleanup_and_exit 2
-            fi
-        fi
 
         if [ $PROVIDER_ONLY -eq 0 ]; then
             check_if_pkg_is_installed omi
             if [ $? -eq 0 ]; then
-                pkg_upd $OMI_PKG omi
-                # It is acceptable that this fails due to the new omi being 
-                # the same version (or less) than the one currently installed.
-                OMI_EXIT_STATUS=0
+                pkg_upd_list $OMI_PKG omi
             else
-                pkg_add $OMI_PKG omi
-                OMI_EXIT_STATUS=$?
+                pkg_add_list $OMI_PKG omi
             fi
         fi
 
-        pkg_add $OM_PKG scx
-        SCX_EXIT_STATUS=$?
+        pkg_add_list $OM_PKG scx
+
+        pkg_upd ${UPD_PKG_QUEUE}
+        # It is acceptable that this fails due to the new omi being 
+        # the same version (or less) than the one currently installed.
+        OMI_EXIT_STATUS=0
```

Comment:
Sorry for confusion. When I said "it'll not catch SCX install issues", I meant that more generically (since the shell bundle ultimately installs SCX with OMI).

Perhaps a more precise way to say what I was trying to say:

"Setting OMI_EXIT_STATUS to zero is just wrong (as it won't catch actual installation problems with the OMI component)."

But I think you figured out what I meant! :smile: